apps/box/src/components/Cards/ConnectedDevicesCard.tsx

```diff
@@ -126,7 +125,7 @@ export const ConnectedDevicesCard = () => {
           </FxText>
         </FxBox>
       ) : ENTRIES.length === 1 ? (
-        <DeviceCard {...ENTRIES[0]} />
+        <DeviceCard data={ENTRIES[0]} />
```

Comment:
could the `<CardCarousel />` comp not be smart and handle the single case? Otherwise we will have to do this repeatedly for every carousel.

Reply:
done